test/basic.js

```diff
@@ -15,6 +15,10 @@ describe('Basic', () => {
     }, 1000);
   });
 
+  it('should only test in the correct environment', function(done) {
```

Comment:
this `done` could be removed too.

Reply:
b5e7da9506907f8c417a4cf2fc20f9275aaac5c9
I have removed it.